build/ferretdb.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=FerretDB Database
+Documentation=https://docs.ferretdb.io/
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=/usr/bin/ferretdb
+Restart=on-failure
+
+# Configure the FerretDB service with `systemctl edit ferretdb`.
+# For more configuration options check https://docs.ferretdb.io/configuration/flags/
+
+Environment="FERRETDB_POSTGRESQL_URL=postgres://username:password@127.0.0.1:5432/ferretdb"
```

Comment:
Let's use the default value for that one, but keep it to serve as an example

Reply:
That's not the default value

Reply:
Ah sorry, I've used the one from all-in-one dockerfile